ml-agents/mlagents/trainers/poca/optimizer_torch.py

```diff
@@ -95,6 +95,12 @@ def baseline(
                 memories=memories,
                 sequence_length=sequence_length,
             )
+
+            num_agents = self.network_body.agent_count(
```

Comment:
I wonder if we should just add this to the buffer and calculate it when processing trajectories?